### PR TITLE
[SD-5403]-(fix)- Highlighted items stay highlighted after search term…

### PR DESCRIPTION
**Pull request by sjunaid**
… is removed
included fix for [SD-5399](https://dev.sourcefabric.org/browse/SD-5399) as well

### CODE REVIEW

**Comment by mdhaman** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

File: scripts/superdesk-search/services/SearchService.js [hunk 1]
```diff
@@ -448,6 +448,7 @@ export function SearchService($location, gettext, config, session) {
                     _.forEach(_.keys(item.es_highlight), function (key) {
                         item[key] = item.es_highlight[key][0];
                     });
+                    item.reRender = true; // used to make it re-render.
```
Comment (petrjasek) [inline scripts/superdesk-search/services/SearchService.js]: imo would be better to create a new item here with `return angular.extend({}, item);`, rather then adding here a flag and create this new item later.

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**